src/openms/source/FORMAT/QcMLFile.cpp

```diff
@@ -1038,8 +1043,1007 @@ namespace OpenMS
     }
   }
 
-  void QcMLFile::store(const String& filename) const
+  float calculateSNmedian(const MSSpectrum& spec, bool norm = true)
```

Comment:
can you check if SignalToNoiseEstimatorMedian
would be a good replacement?
FILTERING/NOISEESTIMATION/SignalToNoiseEstimatorMedian.h

Reply:
seems to be too far of. probably better to move this method into a QCBase derived metric at some point